packages/react/src/components/RadioButton/RadioButton.stories.mdx

```diff
@@ -81,6 +87,19 @@ export const Template = (args = {}) => (
   <RadioButton {...{ ...defaultArgs, ...args }} />
 );
 
+export const SiblingLabelTemplate = (args = {}) => (
+  <div className={classes.parentDiv}>
+    <label
+      htmlFor='radio-button-id'
+      className={classes.label}
+    ></label>
+    <RadioButton
+      {...{ ...defaultArgs, ...args }}
+      radioId='radio-button-id'
+    />
+  </div>
+);
```

Comment:
hm, I think that the following code looks a little hard to read. If I'm a new developer that needs to use the RadioButton component, it does not feel natural to `renderLabelAsSpan` and have an empty label outside of the RadioButton.

Your reason for implementing this is that you want the opportunity to make a clickable card to focus the radio button. What about refactoring the RadioButton component to have the availability to receive the "ref"? Then you can access the radio button from the parent component and focus by doing something like this: `radioButtonRef.current.focus()`. 

Another benefit of using the ref instead of the "renderLabelAsSpan" is that we are moving the code in the right direction due to the new radio button API which will support the ref. :)

```suggestion
export const SiblingLabelTemplate = (args = {}) => (
  <div className={classes.parentDiv}>
    <label
      htmlFor='radio-button-id'
      className={classes.label}
    ></label>
    <RadioButton
      {...{ ...defaultArgs, ...args }}
      radioId='radio-button-id'
    />
  </div>
);
```